scripts/apps/search/components/fields/takekey.tsx

```diff
@@ -1,16 +1,14 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import {gettext} from 'core/ui/components/utils';
 
 export const takekey: React.StatelessComponent<any> = (props) => {
-    const {gettextCatalog} = props.svc;
-
     if (props.item.anpa_take_key) {
         return React.createElement('span', {className: 'takekey', key: 'takekey'},
-            gettextCatalog.getString(props.item.anpa_take_key));
+            gettext(props.item.anpa_take_key));
```

Comment:
Do not use `gettext` for dynamic values. Instead an alias of `gettext` should be used so the code inside `gettext` isn't extracted and sent to Transifex.

It doesn't look like it makes sense to translate this at all.

Reply:
Not sure what values we can have for anpa_take_key